src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -108,19 +118,21 @@
      *            - for updating badge information.
      */
     @Inject
-    public GameboardsFacade(final PropertiesLoader properties, final ILogManager logManager,
-            final GameManager gameManager, final QuestionManager questionManager, final UserAccountManager userManager,
-            final UserAssociationManager associationManager,
-                            final UserBadgeManager userBadgeManager) {
+    public GameboardsFacade(final SegueContentFacade api, final PropertiesLoader properties, final ILogManager logManager,
+                            final GameManager gameManager, final QuestionManager questionManager,
+                            final UserAccountManager userManager, final UserAssociationManager associationManager,
+                            final UserBadgeManager userBadgeManager, @Named(CONTENT_INDEX) final String contentIndex) {
         super(properties, logManager);
 
+        this.api = api;
```

Comment:
It is odd to have a facade depending on another facade, is there not a manager that has a find matching content method?

Reply:
It looks like that method just shouldn't be in the facade. We should move findMatchingContent into the contentManager.

Reply:
Do you mean that we should move `findMatchingContent` from `SegueContentFacade` to `GitContentManager` or that the call to `findMatchingContent` should be made from `GitContentManager`?

Reply:
The first one. Or you can use the manager method which that method calls - contentManager.findByFieldNames(...)